json-schema-generator/src/main/java/io/micronaut/jsonschema/generator/loaders/UrlLoader.java

```diff
@@ -124,7 +116,15 @@ public static void setAllowedUrlPatterns(List<String> allowedUrlPatterns) {
         UrlLoader.allowedUrlPatterns = allowedUrlPatterns;
     }
 
-    public static void addAllowedUrlPatterns(String allowedUrlPattern) {
```

Comment:
this is a breaking change. We are renaming a `public` method.

Reply:
@elifKurtay confirmed me never shipped a version of `micronaut-json-schema` with `UrlLoader::addAllowedUrlPatterns(String)`.